scene/main/viewport.cpp

```diff
@@ -1277,21 +1277,24 @@ void Viewport::_gui_show_tooltip() {
 	gui.tooltip_popup->child_controls_changed();
 }
 
-void Viewport::_gui_call_input(Control *p_control, const Ref<InputEvent> &p_input) {
+bool Viewport::_gui_call_input(Control *p_control, const Ref<InputEvent> &p_input) {
+	bool stopped = false;
 	Ref<InputEvent> ev = p_input;
 
-	// Mouse wheel events can't be stopped.
+	// Returns true if an event should be impacted by a control's mouse filter.
+	bool is_mouse_event = false;
```

Comment:
Not very important but you could do:
```suggestion
	bool is_mouse_event = Ref<InputEventMouse>(p_input).is_valid();
```
so you don't need the `else if`.